winui/Ribbon/Ribbon-QAT.md

```diff
@@ -121,6 +121,20 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
 
 ![WinUI Ribbon QAT menu items](Ribbon-QAT-images/winui-ribbon-qat-menu-items.png)
 
+## Add items to QAT customize dialog window
+
+To open QAT Customize Window, select `MoreCommands` option from the ContextMenu of the QAT. In the QAT Customized dialog Window, the list of Commands is available. The Commands can be filtered only from the Particular tab by using `Choose commands from` option. Then, select the Command to add to the QuickAccessToolBar and add commands to the right Pane of the Quick Access ToolBar Dialog by clicking Add Button. Finally click OK.
+
+![Adding WinUI Ribbon Items to QAT Customize Window](Ribbon-QAT-images/winui-ribbon-items-in-more-commands.png)
```

Comment:
![MoreCommands option in QAT drop-down context menu] (Ribbon-QAT-images/winui-ribbon-more-commands-option.png)

Modify image name and content as per the above suggestions.

Reply:
updated